src/core.ts

```diff
@@ -191,8 +191,8 @@ export const isolate = <M>(ctr: Container<M>, opts: any = {}): IsolatedContainer
  * will be treated as an updater.
  */
 export function seq<M>(...updaters: Updater<M>[]) {
-  return function (model: M, msg?: GenericObject, relay?: GenericObject): UpdateResult<M> {
-    const merge = ([{}, cmds], [newModel, newCmds]) => [newModel, flatten(cmds.concat(newCmds))];
+  return function(model: M, msg: GenericObject, relay: GenericObject): UpdateResult<M> {
+    const merge = ([{ }, cmds], [newModel, newCmds]) => [newModel, flatten(cmds.concat(newCmds))];
```

Comment:
The spacing here is inconsistent with the coding standard. Should be `return function (` and `merge = ([{}`.